docs/google-cloud.md

```diff
@@ -155,7 +155,7 @@ Now that you have a challenge up and running, you need to test it to make sure i
 Run the following command to connect to your challenge:
 
 ```
-telnet $(make ip) 1
```

Comment:
netcat is not installed in cloudshell by default

Reply:
ok, reverted. Though it's not ideal since telnet has control characters so it will break for anything that is not ascii only